Services/CO.CDP.DataSharing.WebApi/AutoMapper/DataSharingProfile.cs

```diff
@@ -9,25 +15,125 @@ public class DataSharingProfile : Profile
+
+        CreateMap<Persistence.SharedConsent, SupplierInformationData>()
+            .ForMember(m => m.Form, o => o.MapFrom(m => m))
+            .ForMember(m => m.AnswerSets, o => o.MapFrom(m => m.AnswerSets))
+            .ForMember(m => m.Questions,
+                o => o.MapFrom(m => m.AnswerSets.SelectMany(a => a.Section.Questions).DistinctBy(q => q.Id)));
+
+        CreateMap<Persistence.SharedConsent, Model.Form>()
+            .ForMember(m => m.Name, o => o.MapFrom(m => m.Form.Name))
+            .ForMember(m => m.SubmissionState, o => o.MapFrom(m => m.SubmissionState.ToString()))
+            .ForMember(m => m.SubmittedAt,
+                o => o.MapFrom(m => m.SubmittedAt.HasValue ? m.SubmittedAt.Value.DateTime : default))
+            .ForMember(m => m.OrganisationId, o => o.MapFrom(m => m.Organisation.Guid))
+            .ForMember(m => m.FormId, o => o.MapFrom(m => m.Guid))
+            .ForMember(m => m.FormVersionId, o => o.MapFrom(m => m.Form.Version))
+            .ForMember(m => m.IsRequired, o => o.MapFrom(m => m.Form.IsRequired))
+            .ForMember(m => m.ShareCode, o => o.MapFrom(m => m.ShareCode));
+
+        CreateMap<CO.CDP.OrganisationInformation.Persistence.Forms.FormAnswerSet, Model.FormAnswerSet>()
+            .ForMember(m => m.Id, o => o.MapFrom(m => m.Guid))
+            .ForMember(m => m.Answers, o => o.MapFrom(m => m.Answers));
+
+        CreateMap<CO.CDP.OrganisationInformation.Persistence.Forms.FormSection, Model.FormSection>()
+            .ForMember(m => m.Id, o => o.MapFrom(m => m.Guid))
+            .ForMember(m => m.Title, o => o.MapFrom(m => m.Title))
+            .ForMember(m => m.Type, o => o.MapFrom(m => m.Type))
+            .ForMember(m => m.FormId, o => o.MapFrom(m => m.FormId))
+            .ForMember(m => m.Form, o => o.Ignore())
+            .ForMember(m => m.Questions, o => o.Ignore())
+            .ForMember(m => m.AllowsMultipleAnswerSets, o => o.MapFrom(m => m.AllowsMultipleAnswerSets))
+            .ForMember(m => m.Configuration, o => o.Ignore());
+
+        CreateMap<CO.CDP.OrganisationInformation.Persistence.Forms.FormAnswer, Model.FormAnswer>()
+            .ForMember(m => m.QuestionName, o => o.MapFrom(m => m.Question.Guid))
+            .ForMember(m => m.BoolValue, o => o.MapFrom(m => m.BoolValue))
+            .ForMember(m => m.NumericValue, o => o.MapFrom(m => m.NumericValue))
+            .ForMember(m => m.StartValue, o => o.MapFrom(m => m.StartValue))
+            .ForMember(m => m.EndValue, o => o.MapFrom(m => m.EndValue))
+            .ForMember(m => m.TextValue, o => o.MapFrom(m => m.TextValue))
+            .ForMember(m => m.OptionValue,
+                o => o.MapFrom(m => !string.IsNullOrWhiteSpace(m.OptionValue) ? int.Parse(m.OptionValue) : default));
+
+        CreateMap<CO.CDP.OrganisationInformation.Persistence.Forms.FormQuestion, Model.FormQuestion>()
+            .ForMember(m => m.Type, o => o.MapFrom(m => (int)m.Type))
```

Comment:
The Type here needs to be returned as a string, but it seems it's sometimes a string, while in other cases it's an int:

```
    {
      "type": "Boolean",
      "name": "c1e2e3f4-5a6b-7c8d-9e0f-123456789abc",
      "title": "The financial information you will need.",
      "text": "<p class=\"govuk-body\">You will need to upload accounts or statements for your 2 most recent financial years.</p><p class=\"govuk-body\">If you do not have 2 years, you can upload your most recent financial year.</p><p c
lass=\"govuk-body\">You will need to enter the financial year end date for the information you upload.</p>",
      "isRequired": true,
      "sectionName": "Financial Information",
      "options": []
    },
    {
      "type": 6,
      "name": "a5b6c7d8-9e0f-1a2b-3c4d-56789ef01234",
      "title": "Check your answers",
      "text": null,
      "isRequired": true,
      "sectionName": "Financial Information",
      "options": []
    },
```

Reply:
I have updated CO.CDP.DataSharing.WebApi.Model.FormQuestionType to be consistent with CO.CDP.OrganisationInformation.Persistence.Forms.FormQuestionType